tools/buffalo/model/templates.go

```diff
@@ -0,0 +1,63 @@
+package model
+
+import (
+	"html/template"
+
+	"github.com/gobuffalo/flect"
+)
+
+var modelTemplate string = `package models
+
+import (
+	{{- range $i := .Imports }}
+	"{{$i}}"
+	{{- end }}
+)
+
+// {{ properize .Name }} model struct
+type {{ properize .Name }} struct {
+	{{- range $attr := .Attrs }}
+	{{ pascalize $attr.Name }}	{{$attr.GoType }} ` + "`" + `json:"{{ underscore $attr.Name }}" db:"{{ underscore $attr.Name }}"` + "`" + `
+	{{- end }}
+}
+
+// {{ pluralize .Name }} array model struct of {{ properize .Name }}
+type {{ pluralize .Name }} []{{ properize .Name }}
+
+// String converts the struct into a string value
+func ({{ .Char }} {{ properize .Name }}) String() string {
+	js, err := json.Marshal({{ .Char }})
+	if err != nil {
+		log.Fatal(err)
```

Comment:
^^ this one is not good :D. Lets throw JSON and do:
```
return fmt.Sprintf("%+v\n", {{.Char}})
```

Reply:
Got it. Thanks for this, i'll keep it on mind for future references.